pkg/api/types.go

```diff
@@ -1391,11 +1391,9 @@ const (
 	// Enforced by the scheduler and Kubelet.
 	// TaintEffectNoScheduleNoAdmit TaintEffect = "NoScheduleNoAdmit"
 	// NOT YET IMPLEMENTED. TODO: Uncomment field once it is implemented.
-	// Do not allow new pods to schedule onto the node unless they tolerate the taint,
-	// do not allow pods to start on Kubelet unless they tolerate the taint,
-	// and evict any already-running pods that do not tolerate the taint.
-	// Enforced by the scheduler and Kubelet.
-	// TaintEffectNoScheduleNoAdmitNoExecute = "NoScheduleNoAdmitNoExecute"
+	// Evict any already-running pods that do not tolerate the taint.
+	// Enforced by Kubelet.
+	TaintEffectNoExecute TaintEffect = "NoExecute"
```

Comment:
what about taint effect Evict/Eviction? I think if pod tolerates taint with effect Evict it means exactly "continue running if already running"